### PR TITLE
Updated item model and added a getBoardsbyBook route

### PR DESCRIPTION
**Pull request by dianaestrada417**

### CODE REVIEW

**Review by Muskan-R-S — COMMENTED** [review]
This looks good to me!

File: backend/controllers/board.js [hunk 1]
```diff
@@ -66,6 +67,25 @@ boardRouter.get('/getBoard', async (req,res) => {
...
 
 
+boardRouter.get('/getBoardsbyBook', async (req, res) => {
+    try{
+        const {userId, bookTitle, bookAuthor} = req.query;
+        const boards = await Board.find({
+            userId: { $ne: userId }, // Exclude boards that belong to the user
+            bookTitle: bookTitle,
+            bookAuthor: bookAuthor,
+            visibility: true
```
Comment (kharris29) [inline backend/controllers/board.js:77]: could you update the "visibility" field to be "publicVisibility"? I think it's a little more clear that way

File: backend/controllers/board.js [hunk 2]
```diff
@@ -66,6 +67,25 @@ boardRouter.get('/getBoard', async (req,res) => {
 });
 
 
+boardRouter.get('/getBoardsbyBook', async (req, res) => {
```
Comment (kharris29) [inline backend/controllers/board.js:70]: please update camel case: getBoardsByBook

**Review by kharris29 — COMMENTED** [review]
couple tiny comments!

**Pull request merged**